x/evm/keeper/erc20.go

```diff
@@ -26,6 +26,14 @@ const (
 	Erc20GasLimitExecute uint64 = 200_000
 )
 
+// getCallGas returns the gas limit for a call to an ERC20 contract following 63/64 rule (EIP-150)
+// protection against recursive calls ERC20 -> precompile -> ERC20.
+func getCallGasWithLimit(ctx sdk.Context, gasLimit uint64) uint64 {
+	availableGas := ctx.GasMeter().GasRemaining()
+	callGas := (availableGas * 63) / 64
```

Comment:
For technical accuracy, let's make this `N - floor(N/64)`.

Reference: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-150.md